kubernetes/schema_cron_job_spec_test.go

```diff
@@ -0,0 +1,23 @@
+package kubernetes
```

Comment:
Let's put this unit test into validators_test.go rather than having its own file.

Reply:
Also need to run go fmt on this file.

Reply:
@jrhouston thanks should be fixed now